src/index.ts

```diff
@@ -119,6 +108,25 @@ app.listen(port, () => {
 //section for create new question endpoint
 
 //delete question endpoint section
-app.delete('/questions/delete', (req: Request, res: Response) => {
-	res.send(`server got a delete request`);
+app.delete('/questions/:id', async (req: Request, res: Response) => {
+	console.log(parseInt(req.params.id));
+	try {
+		let id = req.params.id;
+		let deleteData = await fsPromises.readFile(library, 'utf8');
+		let jsondeletedata = JSON.parse(deleteData);
```

Comment:
Please can you use camelCase for variable names? i.e. `jsonDeleteData`

Reply:
corrected